### DISCUSSION TITLE
Thank You!

### DISCUSSION THREAD

**Discussion by volosied** [TomNisbet/TommyPROM#27] [General]
Started working on Ben Eater's "Build a 6502 Computer" kit by first creating an Arudino EEPROM Programmer. Could NOT write to the EEPROM for the life of me. Checked every wire 10 times. Tried Ben Eater's code. Nothing worked. Searched around, and  I found this repository.  The `unlock-ben-eater-hardware.ino` script worked like a charm!

Thank you so much!   I can now move onto the next part of the project. :)

**Comment by TomNisbet** [comment]
I'm glad it helped!  Those Ben Eater videos are great.  I built the 8-bit computer (see my nqsap project) and will probably do a 6502 or a VGA interface next.  If you haven't seen it already, there is a really good [beneater group on reddit](https://www.reddit.com/r/beneater/) with lots of helpful posts.  

Good luck with your build.
